SoS/Readme/SoS-Changelog.md

```diff
@@ -1,3 +1,11 @@
+#Version 2.1.3 (November 14, 2024)
+
+- changed spell installation to ADD_SPELL
```

Comment:
Could you please add line indicating what exactly was changed in Alternative Soubar area ARE file?